pilot/pkg/features/pilot.go

```diff
@@ -753,6 +753,10 @@ var (
 
 	EnableOptimizedServicePush = env.RegisterBoolVar("ISTIO_ENABLE_OPTIMIZED_SERVICE_PUSH", true,
 		"If enabled, Istiod will not push changes on arbitraty annotation change.").Get()
+
+	InformerWatchNamespace = env.Register("ISTIO_WATCH_NAMESPACE", "",
```

Comment:
Any chance we can make it a list ( which may have a single element as well ) ? Just for consistency and to not have another change in few months.

Reply:
if we add that support it would just be a csv we split I think, no change needed to the user facing variable? lmk if that isn't what you meant

Reply:
Yes. Maybe an "S" at the end, but we can live with the singular. 

It would still use per-namespace filtered watches - maybe add to the doc that this is intended for either low-privilege OR massive clusters. 

On the interface side - if I remember correctly we have a MeshConfig.configSources list option, one of them is 
a source in k8s://... form. Would it be possible to extend that option with k8s://ns/FOO ? It would also allow expressing 'watch a different cluster', etc. And we already have it.